api/CreateMonitor.json

```
@@ -0,0 +1,84 @@
+{
+  "api": {
+    "CreateMonitor": {
+      "__compat": {
+        "spec_url": "https://webmachinelearning.github.io/writing-assistance-apis/#createmonitor",
+        "support": {
+          "chrome": {
+            "version_added": "138"
+          },
+          "chrome_android": "mirror",
+          "deno": {
+            "version_added": false
+          },
+          "edge": "mirror",
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": "mirror",
+          "ie": {
+            "version_added": false
+          },
+          "nodejs": {
+            "version_added": false
+          },
+          "oculus": "mirror",
+          "opera": "mirror",
+          "opera_android": "mirror",
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": "mirror",
+          "samsunginternet_android": "mirror",
+          "webview_android": "mirror",
+          "webview_ios": "mirror"
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "ondownloadprogress": {
+        "__compat": {
```

We usually record the event, rather than the event handler property:

```suggestion
      "downloadprogress_event": {
        "__compat": {
          "description": "`downloadprogress` event",
```

See: https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/api.md#dom-events-eventname_event

---

LGTM; thanks @caugner.